### DISCUSSION TITLE
Connect to wifi using EAP (SSID, Username, Password)

### DISCUSSION THREAD

**Discussion by iotricity** [earlephilhower/arduino-pico#724] [Q&A] [answered]
Is it possible to connect the RPi 2040W to an enterprise wifi network that requires a username as well? To join the wifi, EAP needs a username and password to connect. The current options are only ussing SSID or SSID and password.

Thanks in advance and kind regards.
Jeroen

**Comment by earlephilhower** [comment, marked as answer]
No, the CYW43 driver does not support EAP right now so the core can't, either.

**Reply by iotricity** [reply]
Thanks for your quick reply.
